### ISSUE TITLE
Naming conventions

### ISSUE THREAD

**Issue by kitebuggy** [adimichele/hubspot-ruby#97]
This is bordering on extreme pedantry, but v1.0 gives us the chance to standardise on the official HubSpot naming conventions in this code.

For example, Hubspot => HubSpot as a module name.

Also, the HubSpot term for the company identity is companyID (rather than vid, which typically refers to the contact ID).

Happy to submit a PR if this something worth considering at this forthcoming release?

**Comment by cbisnett** [comment]
Yeah, I agree it would be nice to get some of these things ironed out for v1.0. If you want to create a pull-request based off master that would be great.

Since most of these will be breaking changes, can you also add a section to the readme that details the changes needed to upgrade from v0.x to v1.0. This way we can make it easy for existing users to upgrade to the new version without making them figure out what they need to do. Thanks!

**Comment by kitebuggy** [comment]
Hi @cbisnett,

Moved to HubSpot as discussed, but included deprecations for the older calls and classes, to allow for a less painful transition.  See PR 128: https://github.com/adimichele/hubspot-ruby/pull/128

I'd welcome feedback on suitability, approach, etc., please.

Thanks,

Jason
(P.S. sorry it took so long - work keeps getting in the way of the fun stuff!)
